cmd/oras/internal/option/remote.go

```diff
@@ -191,6 +195,14 @@ func (opts *Remote) tlsConfig() (*tls.Config, error) {
 			return nil, err
 		}
 	}
+	if opts.CertFilePath != "" && opts.KeyFilePath != "" {
```

Comment:
What if user only provide partial input? Like only one cert or one key file is provided? Should we add error check to suggest user provide both?

Helm seems silently ignore partial input, which I think ORAS should enhance.

Reply:
@FeynmanZhou

Reply:
```suggestion
	if opts.CertFilePath != "" || opts.KeyFilePath != "" {
```

I would expect it to work like that^, but I'm not sure how the LoadX509KeyPair works

Reply:
@TerryHowe this option that we are enabling is for client/mtls auth. so we would expect to have both a certificate and key.

@TerryHowe @qweeah i did look into ways of enforcing both options are provided and we can look to leverage the `MarkFlagsMutuallyExclusive` of Cobra.

https://github.com/spf13/cobra/blob/main/site/content/user_guide.md#flag-groups

The downside is that it would require a large number of changes since these are global options. I could emit an error in the `tlsConfig` function, but i would rather perform the validation early

Let me know what path I should go down.

Reply:
`MarkFlagsMutuallyExclusive` returns error when `opts.CertFilePath != "" && opts.KeyFilePath != ""`

Here we need to return error when `opts.CertFilePath != "" XOR opts.KeyFilePath != ""`

> it would require a large number of changes since these are global options.

You just need to add check into the `remote.Parse` and the check will be applied to all commands before execution.

https://github.com/oras-project/oras/blob/202bcf44e210ff51a1b2acf4383f80420ddaa485/cmd/oras/internal/option/remote.go#L142-L158

Reply:
Perfect timing as this was added recently! Will add the check

Reply:
@qweeah Updated to account for the requested changes. Please take a look whenever you have a moment

Reply:
LGTM